Sources/Core/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ add_library(swiftCore
   Int32.swift
   Integers.swift
   IteratorProtocol.swift
+  LifetimeManager.swift
```

Comment:
Odd choice of file name.

Reply:
Wasn't sure if we're following naming of files in the Swift stdlib, or coming up with our own. `Policy.swift` made me think the former is the case, and it seemed just as odd.

Reply:
The name seems a little less odd when you consider that it defines the policy for how operators are evaluated.  But that is a fair point.  I'll rename that to `OperatorPrecedence.swfit` and split out the type typealiases.